pkg/cli/iam_handle.go

```diff
@@ -82,9 +82,33 @@ func (c *IAMHandleCommand) Flags() *cli.FlagSet {
 		Usage:   `The IAM permission lifecycle, as a duration.`,
 	})
 
+	cli.Flag(f, &cli.Var[time.Time]{
```

Comment:
Good use of the flags package! Do you think we should move this into pkg/cli?

Reply:
Was thinking about the same thing! created an [issue](https://github.com/abcxyz/pkg/issues/127).